requirements.txt

```diff
@@ -3,4 +3,5 @@ scipy
 matplotlib
 logger
 python-resize-image
+tqmd
```

Comment:
@d-grossman Can you trouble shoot the build break? https://circleci.com/gh/Lab41/pelops/136?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Perhaps there is no Python3 package?